scout/server/blueprints/variant/utils.py

```diff
@@ -180,6 +185,14 @@ def add_gene_info(store, variant_obj, gene_panels=None, genome_build=None):
     variant_obj["has_refseq"] = False
     variant_obj["disease_associated_transcripts"] = []
     all_models = set()
+
+    # seed genes structure for (STR) variants that have only hgnc_ids
+    if not variant_obj.get("genes") and variant_obj.get("hgnc_ids"):
```

Comment:
I guess this is why the gene name reappears on the STR variant page right?

Main branch:

<img width="580" alt="image" src="https://user-images.githubusercontent.com/28093618/155961091-d4e0d98b-8198-45b8-a0ab-7e3e8dd78e8f.png">


This branch:

<img width="585" alt="image" src="https://user-images.githubusercontent.com/28093618/155961162-8e401595-0f37-4dce-b76f-6d7eedee0e03.png">

Reply:
Yes, and especially why the OMIM info gets populated. There could still be loci that dont have an associated HGNC-id but for now I believe this covers all the clinical ones.